lib/bootloader_setup.pm

```diff
@@ -1347,7 +1347,8 @@ sub prepare_disks {
 
     my $disks = script_output('lsblk -n -l -o NAME -d -e 7,11');
     for my $d (split('\n', $disks)) {
-        script_run "wipefs -a /dev/$d";
+        # Ensure we start from scratch
+        script_run("dd if=/dev/zero of=/dev/$d bs=4M count=10 && sync");
```

Comment:
In general, if it helps, we can use dd here, but I wonder why do we need `sync` call here.
I guess `dd` is available in all cases when we need this, I wonder if `-f` flag of `wipefs` would help too. From my experience running it with force flag was always enough.
So if it helps +1, let me know if you need hints for the VR jobs, as we use this feature on powerVM and zVM setups.

Reply:
We had the same idea with the `-f` flag, however it's difficult to tell if this is gonna solve those occasional failures. In [this fail here](http://openqa.qam.suse.cz/tests/17829#step/boot_from_pxe/12) one can see that `wipefs` is clearing the partition table but somehow [YaST still finds them](http://openqa.qam.suse.cz/tests/17829#step/partitioning_firstdisk/2). It could also be that YaST is more clever and thoroughly than we expect.

`dd` is the NUKE IT FROM ORBIT approach, which should work in all cases. The `sync` can go, indeed.

Shall we perhaps first change it to `wipefs -af` and only if we still see some failures switch to `dd`? It's difficult to tell, I didn't came up with a clean reproducer in this case.

Reply:
I've changed it to `wipefs -af` for now.